plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -112,7 +115,42 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 			waitUntil: 'networkidle',
 		} );
 		await page.locator( '#title' ).fill( nonVirtualProductName );
+
+		await page
+			.frameLocator( '#content_ifr' )
+			.locator( '.wp-editor' )
+			.fill( productDescription );
+
 		await page.locator( '#_regular_price' ).fill( productPrice );
+
+		await page.locator( '.inventory_options' ).click();
+		await page.locator( '#_sku' ).fill( '11' );
+
+		const productDimensions = {
+			weight: '2',
+			length: '20',
+			width: '10',
+			height: '30',
+		};
+
+		await page.locator( '.shipping_options' ).click();
+		await page.locator( '#_weight' ).fill( productDimensions.weight );
+		await page.locator( '#product_length' ).fill( productDimensions.length );
+		await page.locator( '#product_width' ).fill( productDimensions.width );
+		await page.locator( '#product_height' ).fill( productDimensions.height );
+
+		await page
+			.frameLocator( '#excerpt_ifr' )
+			.locator( '.wp-editor' )
+			.fill( productDescriptionShort );
+
+		await page.locator( '#product_cat-add-toggle' ).click();
+		await page.locator( '#newproduct_cat' ).fill( Date.now().toString() );
```

Comment:
As a general rule, we try to clean up anything that's left created in a test.  It's a little extra work, but would you mind putting in a couple of API calls to delete the category and tag created?  Could be at the end of the test, or in an `after()` hook -- whatever makes the most sense to you!

Reply:
Done in [0ae5cd2](https://github.com/woocommerce/woocommerce/pull/43710/commits/0ae5cd219bc6f4fb2f1a17ab835b894d30ea2bfa)